diff --git a/vendor/github.com/tektoncd/pipeline/pkg/pod/entrypoint_lookup_impl.go b/vendor/github.com/tektoncd/pipeline/pkg/pod/entrypoint_lookup_impl.go
index a544d9e6..7a1122e5 100644
--- a/vendor/github.com/tektoncd/pipeline/pkg/pod/entrypoint_lookup_impl.go
+++ b/vendor/github.com/tektoncd/pipeline/pkg/pod/entrypoint_lookup_impl.go
@@ -19,6 +19,7 @@ package pod
 import (
 	"context"
 	"fmt"
+	"runtime"
 
 	"github.com/google/go-containerregistry/pkg/authn"
 	"github.com/google/go-containerregistry/pkg/authn/k8schain"
@@ -68,7 +69,14 @@ func (e *entrypointCache) Get(ctx context.Context, ref name.Reference, namespace
 		return nil, fmt.Errorf("error creating k8schain: %v", err)
 	}
 	mkc := authn.NewMultiKeychain(kc)
-	img, err := remote.Image(ref, remote.WithAuthFromKeychain(mkc))
+
+	// By default go-containerregistry pulls amd64 images.
+	// Setting correct image pull architecture based on the underlying platform
+	var pf = v1.Platform{
+		Architecture: runtime.GOARCH,
+		OS:           runtime.GOOS,
+	}
+	img, err := remote.Image(ref, remote.WithAuthFromKeychain(mkc), remote.WithPlatform(pf))
 	if err != nil {
 		return nil, fmt.Errorf("error getting image manifest: %v", err)
 	}
